### PR TITLE
Added `extract_from_files` function

### LINKED ISSUE #28: Add `extract_from_files` function
This takes a list of strings that are paths to image files and extracts. This would use the `extract_from_file` function but in a loop.

### PR DESCRIPTION
**Pull request by gabb222**
Fixes #28 

This function takes a list of paths to TSPNG files and returns a dictionary of dictionaries. The key is the given path and the value is the associated metadata. A unit test was added to test the function.

### CODE REVIEW

**Comment by gabb222** [comment]
@Theia-Scientific/developers ready for review

File: tspng/extraction.py [hunk 1]
```diff
@@ -13,3 +13,16 @@ def extract_from_file(path, mime_type=MIME_TYPE):
     #load
     dict=json.loads(meta[mime_type])
     return dict
+
+def extract_from_files(paths, mime_type=MIME_TYPE):
+    nested_dict = {}
+    for path in paths:
+        #open
```
Comment (volks73) [inline tspng/extraction.py:20]: Can this be refactored to use the `extract_from_file` function? Like so,

```python
def extract_from_files(paths, mime_type=MIME_TYPE):
    nested_dict = {}
    for path in paths:
        nested_dict[path] = extract_from_file(path, mime_type)
    return nested_dict
```

**Review by volks73 — CHANGES_REQUESTED** [review]
I love the idea of the path being the keys for the dict. Nice touch.

Thank you for adding the test.

I am requesting a change to reduce duplicate code and re-use an existing implementation.

File: tspng/extraction.py (hunk 1 above)
Comment (gabb222) [reply tspng/extraction.py]: Change made as requested, thanks!

**Comment by gabb222** [comment]
@Theia-Scientific/developers ready for review

**Pull request merged**